packages/devtools/lib/src/debugger/debugger.css

```diff
@@ -42,6 +48,29 @@
 
 .debugger-items-list {
     overflow-y: scroll;
+    cursor: pointer;
+}
+
+.open-popup {
+    z-index: 1000;
+    position: absolute;
+    top: 0px;
+    left: 5px;
+    height: 200px;
+    background-color: white;
```

Comment:
this color needs to work with the dark and light themes

Reply:
Thanks updated when I pushed second set of changes.